cypress/integration/search.spec.ts

```diff
@@ -0,0 +1,87 @@
+import * as commands from "../support/commands";
+
+context("search", () => {
+  const projectId = "hnrkfr9g6gxymefc3hto37bgmq3eo86sfckky";
+
+  beforeEach(() => {
+    commands.resetProxyState();
+    commands.onboardUser("cloudhead");
+    commands.createProjectWithFixture();
+    cy.visit("./public/index.html");
+  });
+
+  context("when a project is not yet followed", () => {
+    it("follows the project by clicking the follow button", () => {
+      commands.pick("sidebar", "search").click();
+      commands.pasteInto(["search-input"], `rad:git:${projectId}`);
+      commands.pick("follow-toggle").click();
+      commands
+        .pickWithContent(["undiscovered-project"], projectId)
+        .should("exist");
+
+      // Make sure the search input is cleared after the search.
+      commands.pick("sidebar", "search").click();
+      commands.pick("search-input").should("have.value", "");
+    });
+
+    it("follows the project by pressing the [enter] hotkey", () => {
+      commands.pick("sidebar", "search").click();
+      commands.pasteInto(["search-input"], `rad:git:${projectId}`);
+      cy.wait(400);
```

Comment:
Could you add a comment explaining why we need this?

Reply:
That was actually a debug leftover, removed it here: 1f5137005d4dba1745956e587ce428d6733a3eef.